tests/sentry/mail/test_adapter.py

```diff
@@ -487,8 +489,12 @@ def test_notify_users_with_owners(self):
             user=user2,
             project=project,
         )
-        event_all_users = self.store_event(data=make_event_data("foo.cbl"), project_id=project.id)
-        self.assert_notify(event_all_users, [user.email])
+
+        with self.feature({"organizations:notification-all-recipients": True}):
```

Comment:
There is not problem with this line but did you know:
```py
with self.feature({"organizations:notification-all-recipients": True}):
```
```py
with self.feature(["organizations:notification-all-recipients"]):
```
```py
with self.feature("organizations:notification-all-recipients"):
```
all three of these are equivalent. In the latter two the `True` is implied.